### PR TITLE
Docs in the main repository

### PR DESCRIPTION
**Pull request by brew**
Moved the docs from the gh-pages branch to a subdirectory on the normal master branch. 

Also reformatted the docs so they can be hosted on ReadTheDocs: http://opendatacensus.readthedocs.io/en/latest/

Once this is merged, the CNAME for http://census.okfn.org/ can be changed to point to the readthedocs site.

### CODE REVIEW

File: docs/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+census.okfn.org
```
Comment (pwalsh) [inline docs/CNAME]: This is only needed for gh pages AFAIK.

**Comment by pwalsh** [comment]
@brew 👍

**Comment by pwalsh** [comment]
@brew i've done the domains, and added the canonical domain in readthedocs

**Comment by brew** [comment]
@pwalsh Nice one - I'll merge and have RTD pull from master

**Pull request merged**